src/test/java/codeu/service/BBCodeParserTest.java

```diff
@@ -0,0 +1,64 @@
+package codeu.service;
+
```

Comment:
Remove

Reply:
Do you mean remove the newline or the package statement?

Reply:
Since you asked for a reference to the guide, section 3 mentions spacing for sections of java files ([here](https://google.github.io/styleguide/javaguide.html#s3-source-file-structure))